### PR TITLE
Install junit plugin after openshift-ansible installation

### PR DESCRIPTION
**Pull request by jhadvig**
@stevekuznetsov

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
@jhadvig is there a test job for this?

**Comment by jhadvig** [comment]
@stevekuznetsov yes. [here](https://ci.openshift.redhat.com/jenkins/view/All/job/continuous-upgrade_provision-job/31/consoleFull)

File: continuous-upgrade/generated/continuous-upgrade_provision-job.xml [hunk 1]
```diff
@@ -64,10 +64,11 @@ sslverify = 0
 EOR
 sudo cp ./openshift-int.repo /etc/yum.repos.d
 
+sudo yum --disablerepo=* --enablerepo=openshift-int,oso-rhui-rhel-server-releases install -y atomic-openshift-utils
 cd /data/src/github.com/openshift/aos-cd-jobs/
 git pull origin master
 /data/src/github.com/openshift/aos-cd-jobs/continuous-upgrade/actions/install_junit.sh
-sudo yum --disablerepo=* --enablerepo=openshift-int,oso-rhui-rhel-server-releases install -y atomic-openshift-utils
+sudo yum-config-manager --disable rhel-7-server-ose-3\*,li
```
Comment (stevekuznetsov) [inline continuous-upgrade/generated/continuous-upgrade_provision-job.xml]: ?

Comment (stevekuznetsov) [inline continuous-upgrade/generated/continuous-upgrade_provision-job.xml]: Ah I see

**Comment by stevekuznetsov** [comment]
@jhadvig OK let me know when you are happy

**Pull request merged**